charts/agent/templates/daemonset.yaml

```diff
@@ -95,6 +96,13 @@ spec:
             - name: SYSDIG_BPF_PROBE
               value:
           {{- end }}
+          {{- if (include "agent.universalEbpfEnforced" .) }}
```

Comment:
I think this env is not needed...if `agent.universalEbpfEnforced` returns true we don't start the `initContainer`